internal/metrics/prometheus.go

```diff
@@ -152,6 +154,23 @@ func NewCtrlFuncMetrics() *CtrlFuncMetrics {
 		[]string{SuccessKey},
 	)
 
+	// TODO: add new metric for fallback config generation or add a "fallback" label?
```

Comment:
Separate metric IMHO as we already have a separate set of metrics for that.

Reply:
Added metrics for fallback translation.